src/seedu/addressbook/storage/StorageFile.java

```diff
@@ -113,12 +114,21 @@ public void save(AddressBook addressBook) throws StorageOperationException {
 
     /**
      * Loads data from this storage file.
-     *
+     * 
+     * @return an {@link AddressBook} containing the data in the file, or an empty {@link AddressBook} if it
```

Comment:
Don't think that wording is apporpriate in this context. load() is called during the program start up where there is no existing AddressBook object. It passes in the file with all the saved data and the method should return a new AddressBook object with all the previously saved data in it which is what the header comment is trying to imply.